README.md

```diff
@@ -1,3 +1,47 @@
 # conda-auth
 
-A conda plugin for handling multiple authentication schemes
+A conda plugin for handling various HTTP based authentication schemes.
```

Comment:
```suggestion
A conda plugin for handling various authentication schemes.
```

Let's keep it more general. What if we decided to support more than just HTTP in the future 🤷